app/models/expanded_taxonomy.rb

```diff
@@ -0,0 +1,47 @@
+class ExpandedTaxonomy
```

Comment:
@carvil Rewritten in this form it doesn't actually help much in terms of readability - certainly not relative to the additional logic and naming complexity it introduces. The main issue being that a lot of the hash access you see is the API of a TreeNode rather than a hash from the GdsApiAdapters.

A potential refactor might be to replace `TreeNode` with our own version that provides exactly the same interface but puts `content_id` and `parent_taxons` behind proper methods - rather than the `content` hash provided by `TreeNode`. That feels like a little more than this PR should aim to cover right now, though.